src/concrete/ml/onnx/onnx_utils.py

```diff
@@ -495,3 +507,91 @@ def remove_initializer_from_input(model: onnx.ModelProto):  # pragma: no cover
             inputs.remove(name_to_input[initializer.name])
 
     return model
+
+
+# Remove this function once the truncate feature is released
+# FIXME: https://github.com/zama-ai/concrete-ml/issues/397
+def compute_lsb_to_remove_for_trees(onnx_model: onnx.ModelProto, q_x: numpy.ndarray) -> List[int]:
+    """Compute the LSB to remove for trees in the first and second stage.
```

Comment:
"first and second stage" -> even though this function is specific to tree-based models, these terms are not very clear. Either do not mention them, or then explain a bit what are these stages (could be just a link to the tree-based paper) !

Reply:
there are only two "stages", just remove ` in the first and second stage.` and say something like `Compute the LSB to remove to perform rounded comparisons in trees.`

Reply:
yes that would be great indeed !